### DISCUSSION TITLE
Reverse Proxy Load Balance

### DISCUSSION THREAD

**Discussion by ErfanTech** [XTLS/Xray-core#3035] [Q&A] [answered]
How can i use load balance in my xray-core config(like recently added `Least Load balancer` method https://github.com/XTLS/Xray-core/commit/fa5d7a255b95215916d93bf371aaea55ea758381) ?
I'm using `Reverse Proxy` tunnel between two servers.
@yuhan6665

**Comment by shakibamoshiri** [comment, marked as answer]
sure you can 
https://www.youtube.com/watch?v=uLH4oXCEpBY
